routing/transit_graph.cpp

```diff
@@ -80,6 +80,7 @@ void TransitGraph::Fill(vector<transit::Stop> const & stops, vector<transit::Gat
 
   for (auto const & gate : gates)
   {
+    CHECK_NOT_EQUAL(gate.GetWeight(), transit::kInvalidWeight, ("Gate should have valid weight."));
```

Comment:
Can we use ASSERTions here?

Reply:
Yes, but we'll have invariant violation in release for bad mwm's. We should not have bad mwms but if we have it's much more simple to find the reason of chash if it'll be here.

Reply:
Okay